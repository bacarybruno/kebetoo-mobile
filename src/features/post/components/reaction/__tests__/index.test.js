import setupTest from '@app/config/jest-setup';

import Reaction from '../index';

const givenReaction = setupTest(Reaction)({
  iconName: 'like',
  count: 1100,
  onPress: jest.fn(),
  color: 'reactions',
});

it('renders Reaction', () => {
  const { wrapper } = givenReaction();
  expect(wrapper.toJSON()).toMatchSnapshot();
});
