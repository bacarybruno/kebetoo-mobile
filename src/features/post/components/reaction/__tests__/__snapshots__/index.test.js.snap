// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Reaction 1`] = `
<View
  accessible={true}
  focusable={true}
  hitSlop={
    Object {
      "bottom": 10,
      "left": 5,
      "right": 5,
      "top": 10,
    }
  }
  onClick={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "alignItems": "center",
      "flexDirection": "row",
      "marginRight": 16,
      "minWidth": 35,
      "opacity": 1,
    }
  }
>
  <Text
    allowFontScaling={false}
    style={
      Object {
        "color": "rgba(0,0,0,0.54)",
        "fontFamily": "icomoon",
        "fontSize": 18,
        "fontStyle": "normal",
        "fontWeight": "normal",
        "marginRight": 4,
      }
    }
    text="kebeticon-like"
  />
  <Text
    style={
      Array [
        false,
        false,
        false,
        Object {
          "backgroundColor": "transparent",
          "color": "#000000",
          "fontFamily": "System",
          "fontSize": 12,
          "fontWeight": "400",
          "letterSpacing": 0,
          "lineHeight": 16,
        },
        Array [
          Object {
            "color": "rgba(0,0,0,0.54)",
          },
          Object {
            "backgroundColor": "transparent",
            "fontFamily": "System",
            "fontWeight": "700",
          },
          undefined,
        ],
      ]
    }
  >
    1
  </Text>
</View>
`;
